ui/src/components/ui/device-control-panel/tabs/dashboard-tab.module.css

```diff
@@ -1,11 +1,57 @@
+.dashboardTabContainer {
+  container-type: inline-size;
+}
+
 .dashboardTab {
   overflow: hidden;
+  display: grid;
```

Comment:
I'm ok with either approach - a grid based one or a vertical flexbox with set height and `flex-wrap: wrap`. But we need to comment this place.